app/components/post_textbox/post_textbox.js

```diff
@@ -3,33 +3,39 @@
 
 import React, {PropTypes, PureComponent} from 'react';
 import {
-    Platform,
     TouchableHighlight,
-    View, Text
+    View,
+    Text
 } from 'react-native';
-import Icon from 'react-native-vector-icons/FontAwesome';
+import Icon from 'react-native-vector-icons/Ionicons';
+import ImagePicker from 'react-native-image-crop-picker';
+import {RequestStatus} from 'mattermost-redux/constants';
 
 import Autocomplete from 'app/components/autocomplete';
+import FileUploadPreview from 'app/components/file_upload_preview';
 import FormattedText from 'app/components/formatted_text';
 import TextInputWithLocalizedPlaceholder from 'app/components/text_input_with_localized_placeholder';
 import {changeOpacity} from 'app/utils/theme';
 
-// import PaperClipIcon from './components/paper_clip_icon.js';
-
 const MAX_CONTENT_HEIGHT = 100;
 
-export default class PostTextbox extends PureComponent {
+class PostTextbox extends PureComponent {
```

Comment:
Good catch. I was planning on using injectIntl for the modal_options but instead decided to just pass in the `id` and `defaultMessage` as properties in the options. I'll change it back.